plugins/pulp_rpm/plugins/importers/yum/associate.py

```diff
@@ -102,12 +104,10 @@ def get_rpms_to_copy_by_key(rpm_search_dicts, import_conduit):
         # ignore checksum from updateinfo.xml
         key['checksum'] = None
```

Comment:
What about changing these to be:

```
del key['checksum']
del key['checksumtype']
```

If that change works I think it would be better.